File: packages/vscode-extension/tests/fixtures/textmate/simple-match.go
package main

var a = `//cypher
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
`

var b = `/* cypher */
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
`

var c = "/*cypher*/ MATCH (n:Label) RETURN function(n.property)"

// This one shouldn't highglight
var d = "//cypher MATCH (n:Label) RETURN function(n.property)"

File: packages/vscode-extension/tests/fixtures/textmate/simple-match.fs
let a = "//cypher
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
"

let b = "/* cypher */
    // This is a comment
    MATCH (n:Label) RETURN function(n.property)
"

let c = "/*cypher*/ MATCH (n:Label) RETURN function(n.property)"

// This one shouldn't highglight
let d = "//cypher MATCH (n:Label) RETURN function(n.property)"